daemon/logger/journald/journald.go

```diff
@@ -0,0 +1,31 @@
+package journald
+
+import (
+	"github.com/coreos/go-systemd/journal"
+	"github.com/docker/docker/daemon/logger"
+)
+
+type Journald struct {
+	id string
+}
+
+func New(tag string) (logger.Logger, error) {
+	return &Journald{id: tag}, nil
+}
+
+func (s *Journald) Log(msg *logger.Message) error {
+	jmap := map[string]string{"MESSAGE_ID": s.id}
```

Comment:
Seems like we can save this right in `s` on creation. Loggers is pretty highload stuff, so makes sense to do such micro-optimizations.